pkg/stub/handler.go

```diff
@@ -74,7 +74,74 @@ func (h *Handler) EnsureDefaultClusterIngress() error {
 	return fmt.Errorf("creating default cluster ingress %s/%s: %v", ci.Namespace, ci.Name, err)
 }
 
-func (h *Handler) syncIngressUpdate(ci *ingressv1alpha1.ClusterIngress) error {
+// Reconcile performs a full reconciliation loop for ingress, including
+// generalized setup and handling of all clusteringress resources in the
+// operator namespace.
+func (h *Handler) reconcile() error {
+	// Ensure we have all the necessary scaffolding on which to place router
+	// instances.
+	err := h.ensureRouterNamespace()
```

Comment:
Do we really want to do this for every event? Seems like one time op.

Reply:
Probably not, but it seems advantageous to keep it simple for now. We could do a lookup before bothering to try creating the namespace rather than relying on a superfluous create attempt.